README.md

```diff
@@ -30,6 +30,251 @@ you're a tool author you may be interested in the formal [EBNF grammar][].
 [EBNF grammar]: https://github.com/projectfluent/fluent/tree/master/spec
 
 
+Installation
+------------
+
+python-fluent consists of two packages:
+
+* `fluent.syntax` - includes AST classes and parser. Most end users will not
+  need this directly. Documentation coming soon!
+
+  To install:
+
+        pip install fluent.syntax
+
+
+* `fluent.runtime` - methods for generating translations from FTL files.
+  Documentation below.
+
+  To install:
+
+        pip install fluent.runtime
+
+  (The correct version of ``fluent.syntax`` will be installed automatically)
+
+
+PyPI also contains an old `fluent` package which is an older version of just
+`fluent.syntax`.
+
+Usage
+-----
+
+To generate translations using the ``fluent.runtime`` package, you start with
+the `FluentBundle` class:
+
+    >>> from fluent.bundle import FluentBundle
```

Comment:
One more rename here: `from fluent.runtime`.

Reply:
These ones I was thinking of leaving i.e. currently it is just the package (and things that reflect the package name, like the directory) that are renamed to `fluent.runtime`. Imports are still using `fluent.bundle`, both in the docs and internally.